pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -202,9 +187,73 @@ func (c *Cluster) Start() error {
 	return nil
 }
 
+// reconcile the services while requiring the active mgr to be detected
+func (c *Cluster) ReconcileServices(daemonNameToUpdate string) error {
+	return c.reconcileServices(daemonNameToUpdate, "")
+}
+
+// reconcile the services, if the active mgr is not detected, use the default mgr
+func (c *Cluster) reconcileServices(daemonNameToUpdate, defaultMgr string) error {
+	if daemonNameToUpdate != "" {
+		// If the services are already set to this daemon, no need to attempt to update
+		svc, err := c.context.Clientset.CoreV1().Services(c.clusterInfo.Namespace).Get(context.TODO(), AppName, metav1.GetOptions{})
+		if err != nil {
+			if kerrors.IsNotFound(err) {
+				logger.Warningf("mgr services not found, skipping the update")
+				return nil
+			}
+			logger.Errorf("failed to check current mgr service, proceeding to update... err= %v", err)
```

Comment:
```suggestion
			logger.Errorf("failed to check current mgr service, proceeding to update...%v", err)
```

ditto, can we be more specific, which update?